src/software/ai/hl/stp/play_info.h

```diff
@@ -3,22 +3,68 @@
 #include <string>
 #include <vector>
 
-struct PlayInfo
+/**
+ * A description of the play.  Contains the type of play being
+ * run, its name, and the tactic assignments for all the robots
+ * on the field.
+ */
+class PlayInfo
 {
+   public:
+    /**
+     * Constructs a PlayInfo object.
+     */
+    explicit PlayInfo();
+
+    /**
+     * Get play type
+     *
+     * @return play_type field
+     */
+    std::string getPlayType() const;
+
+    /**
+     * Get play name
+     *
+     * @return play_name field
+     */
+    std::string getPlayName() const;
+
+    /**
+     * Get vector containing tactic assignments for robots
+     *
+     * @return robot_tactic_assignment field
+     */
+    std::vector<std::string> getRobotTacticAssignment() const;
+
+    /**
+     * Sets the play_type field to new_play_type
+     * @param new_play_type, the new play type to be set to
+     */
+    void setPlayType(std::string new_play_type);
+
+    /**
+     * Sets the play_name field to new_play_name
+     * @param new_play_name, the new play name to be set to
+     */
+    void setPlayName(std::string new_play_name);
+
+    /**
+     * Sets the robot_tactic_assignment field to rta
+     * @param rta, the new vector containing robot assignments
+     */
+    void setRobotTacticAssignment(std::vector<std::string> rta);
```

Comment:
Maybe we should rename this function to a plural. 'setRobot' seems like it only affects 1

Reply:
Probably also want to change this to `std::unordered_set` as well.